lib/datasource/conan/index.ts

```diff
@@ -0,0 +1,80 @@
+import { logger } from '../../logger';
+import { cache } from '../../util/cache/package/decorator';
+import { ensureTrailingSlash, joinUrlParts } from '../../util/url';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
+import { conanDatasourceRegex, datasource, defaultRegistryUrl } from './common';
+import type { ConanJSON } from './types';
+
+export class ConanDatasource extends Datasource {
+  static readonly id = datasource;
+
+  override readonly defaultRegistryUrls = [defaultRegistryUrl];
+
+  override readonly caching = true;
+
+  override readonly registryStrategy = 'merge';
+
+  constructor() {
+    super(ConanDatasource.id);
+  }
+
+  private async lookupConanPackage(
```

Comment:
Do we really need this additional function? We we inline this instead?

Reply:
inline is fine at this point.  This was shared during a previous iteration.